src/modules/app/StoreActionType.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import type { StartFetchActionType } from './actions/startFetchAction'
+import type { FinishFetchActionType } from './actions/finishFetchAction'
+import type { SetUiDirectionActionType } from '../i18n/actions/setUIDirection'
+
+export type SetLanguageActionType = { type: 'SET_LANGUAGE', payload: string }
+
+export type SetCurrentCityActionType = { type: 'SET_CURRENT_CITY', payload: string }
+
+export type ToggleDarkModeActionType = { type: 'TOGGLE_DARK_MODE' }
+
+export type StoreActionType =
+  StartFetchActionType<*>
```

Comment:
Do we already have a Payload type which covers all our Payloads?

Reply:
You mean the data in the Payload? No currently not, should we introduce one?